Homerwork-7/Homerwork-7/ViewController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  ViewController.swift
+//  Homerwork-7
+//
+//  Created by Kirill Skrytskii on 24.03.22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        
+        let firstStudent = Student(name: "DrDre", surname: "Pay", numberСlass: 10, subjectAssessment: [("Maths", 5), ("English", 5)])
+        let secontStudent = Student(name: "Smash", surname: "Drake", numberСlass: 11, subjectAssessment: [("Maths", 4), ("English", 4)])
+
+        
+        let studentArray = [firstStudent, secontStudent]
+        
+        let school = School(student: studentArray)
+        school.info()
+        
+    }
+}
+    
+class School {
+    var student:  [Student]
```

Comment:
It's better to call array using plural words because you store not one but several elements: `students`

Please, leave only one space after `:`